iroh/src/commands.rs

```diff
@@ -257,6 +336,52 @@ pub enum Commands {
         #[clap(long, default_value_t = false)]
         single: bool,
     },
+    /// Download data to the running provider's database and provide it.
+    ///
+    /// In addition to downloading the data, you can also specify an optional output directory
+    /// where the data will be exported to after it has been downloaded.
+    Share {
```

Comment:
Is there a way to share sth that I have already downloaded (into the iroh db/store)?

Reply:
If it is already in the store it is already shared. You mean you want to export something you already have without providing a peer to download it from?

Reply:
I am thinking of this flow 

- `iroh get foobar`
- some time passes and someone asks me to share `foobar`
- `iroh ??? foobar` (what do I do here? is this just provide, but with what if I only stored it in irohs db? or is that not an option?)

Reply:
Ah, ok. iroh get is either directly downloading or creating an ephemeral node in process. It does not even talk via rpc. So it is completely separate, and there is nothing you can do except call `iroh add` on the downloaded folder.

We should rethink the ergonomics of the whole get vs. share thing at some point, but for now that is how it works.

Reply:
okay, thanks